wgsl/index.bs

```diff
@@ -10541,196 +10541,452 @@ See [[#function-calls]].
 ## Logical Built-in Functions ## {#logical-builtin-functions}
 
 <table class='data'>
-  <thead>
-    <tr><th>Parameterization<td style="width:40%;">Overload<td>Description
-  </thead>
-  <tr algorithm="vector all">
+  <tr>
+    <th scope="row" style="width:10%">Parameterization
     <td>
-    <td>`@const fn`<br>`all`(|e|: vec|N|&lt;bool&gt;) -> bool
+  <tr algorithm="vector all">
+    <th scope="row">Overload
+    <td><xmp highlight=rust>@const fn all(|e|: vec|N|<bool>) -> bool</xmp>
```

Comment:
These don't render correctly as formulae anymore. The `|e|` were previously selectable to highlight all `e` in that row. Now it renders directly as `all(|e| : vec|N|<bool>) -> bool` instead of `all(e : vecN<bool>) -> bool` (minus the italics).

Reply:
I will remove the `|`s to make it render correctly, but I don't think I can make it syntax highlighted and selectable. I find the syntax highlighting make it easier to read so I'm going to go with that for now.